staging/src/k8s.io/apiserver/pkg/server/filters/timeout_test.go

```diff
@@ -348,9 +354,15 @@ func TestErrConnKilledHTTP2(t *testing.T) {
 		t.Fatal("expected to receive an error")
 	}
 
-	// we should only get one line for this, not the big stack from before
+	// we expect to see one line for this, not the big stack from before,
```

Comment:
this comment is also no longer correct.

Reply:
I fixed the flake, instead of relying on the number of lines in the log, the test now looks for specific patterns to figure out if the stack trace is logged.